tools/gke/kubernetes_api.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python2.7
```

Comment:
both big_query_utils.py and kubernetes_api.py are actually very generic and can be used by other tests as well. Do you recommend I move them both under tools/cloud" ?